### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by casperdcl**
Can the organisation [github.com/tqdm](https://github.com/tqdm/) be granted full maintainer access to the [conda-forge/tqdm-feedstock](https://github.com/conda-forge/tqdm-feedstock/) repo please? Thanks.

- [x] bump version
- [x] update sha256
- [x] fix [badge](https://anaconda.org/conda-forge/tqdm/badges) stuck on displaying 4.8.1 due to #5
- [x] fix conda-forge-linter
- [x] fix travis CI
- [x] appveyor CI

### CODE REVIEW

**Comment by casperdcl** [comment]
Hi @pmlandwehr could you help with fixing CI?

File: recipe/meta.yaml
Comment (pmlandwehr) [inline recipe/meta.yaml:2]: Might as well bump to 4.10.0, since that's the new version (see https://pypi.org/project/tqdm/)

File: recipe/meta.yaml
Comment (pmlandwehr) [inline recipe/meta.yaml:11]: sha for `4.10.0`: `1690ef8d0e06366ad7c8767b9a6408f116a16ff3c0972e05d8c3ac1c2411f0bf`

File: recipe/meta.yaml
Comment (pmlandwehr) [inline recipe/meta.yaml:44]: As far as I know, we can only add maintainers one-at-a-time, not per organization (cc @jakirkham) @lrq3000 are you also interested in being added?

File: recipe/meta.yaml
Comment (pmlandwehr) [inline recipe/meta.yaml:40]: Please add this line: `license_file: LICENSE` so that we point to the license file (hooray for this!)

**Comment by pmlandwehr** [comment]
Hey @casperdcl! Thanks for the patches. I'm going to hold off on fixing the CIs because I have a few tweaks for your patches. Also, please try regenerating the feedstock to see if that sorts out the issues: `conda smithy regenerate` in the top level directory, and then committing any changed files.

**Review by pmlandwehr — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml
Comment (casperdcl) [reply recipe/meta.yaml:2]: I know (I released it) but wanted to make sure v4.9 was also up on conda in case people wanted it. will bump again to 4.10 (and an imminent 4.11) after this PR

File: recipe/meta.yaml
Comment (casperdcl) [reply recipe/meta.yaml:40]: nice

File: recipe/meta.yaml
Comment (casperdcl) [reply recipe/meta.yaml:44]: but can you add the tqdm org as a maintainer of conda-forge/tqdm-feedstock directly in github settings for this repo?

**Comment by casperdcl** [comment]
Ok I've merged #7 and #8 into this PR. once this is merged and appears on conda-forge, will bump to 4.10

**Review by casperdcl — COMMENTED** [review]
ready for merge

**Comment by lrq3000** [comment]
Yes I'm willing to help to maintain the conda package, I just need some time to read about it because I have no idea how the conda packaging and distribution works :/

**Comment by lrq3000** [comment]
Thanks @casperdcl for making this happen, that's great :)

File: recipe/meta.yaml
Comment (pmlandwehr) [reply recipe/meta.yaml:44]: I can, but that would go against the general conda-forge policy as I understand it. No problem with just adding each member one at a time or in groups though, and @lrq3000 seems fine with it. Go ahead and add him too, and I'll merge once the CIs finish.

Comment (ocefpaf) [reply recipe/meta.yaml:44]: @casperdcl we do not handle organizations in our bot scripts making it hard to do what you ask. Is it complicated for you to add your crew via individual GitHub handles?

**Comment by lrq3000** [comment]
I guess Casper's proposition of adding the organization was more about
being future-proof, as the members may change (as it has in the past),
rather than because of a technical limitation. But if that's not possible,
then well if we can at least put a few members as maintainers it should be
ok (it's unlikely both will become unavailable at the same time).

2016-11-14 19:24 GMT+01:00 Filipe notifications@github.com:

File: recipe/meta.yaml
Comment (casperdcl) [reply recipe/meta.yaml:44]: @pmlandwehr @ocefpaf no problem, as @lrq3000 said it just seems more future proof (and probably more morally and legally correct) to support adding the organisation, but we're happy with individual access, thanks.

Comment (ocefpaf) [reply recipe/meta.yaml:44]: > but we're happy with individual access, thanks.

It does not mean we are against it. We just do not have the manpower to check the what needs to be done in the bot scripts to handle that. (Maybe it is a low hanging fruit.)

**Comment by lrq3000** [comment]
It's just the recipe maintainers so morally and legally it's correct even
if there's only your name, it's not the program's developers :)

2016-11-14 20:04 GMT+01:00 Casper da Costa-Luis notifications@github.com:

**Comment by pmlandwehr** [comment]
kk, gonna merge, can add @lrq3000 with the next version tick.

**Pull request merged**
